[Alerting] Enabling import of rules and connectors (#99857)
elastic/kibana: docs/management/action-types.asciidoc

=== Importing and exporting connectors

To import and export rules, use the <<managing-saved-objects, Saved Objects Management UI>>.

> After the succesful import the proper banner will be displayed:

Reviewer: ```suggestion
After a successful import, the proper banner is displayed:
```

[role="screenshot"]

> image::images/coonectors-import-banner.png[Connectors import banner]

Reviewer: Maybe use width=50%.  See comment below.

> Connectors with the missing user sensitive information due to the import will be shown with the 'Fix' button in the list:

Reviewer: ```suggestion
If a connector is missing user sensitive information because of the import,  a **Fix** button appears in the list view.
```
